app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Alcohol Wholesaler Registration Scheme (AWRS) cancellation") {
+    <p @pStyle>Your request to cancel your AWRS registration for @{params("organisationName")} was received on @{params("submissionDate")}.</p>
+<p @pStyle>What happens next</p>
+    <p @pStyle>Your details on the AWRS register will be changed to show that your business has deregistered on the 12 December 2016.</p>
```

Comment:
Thanks for this, the original template had the date hardcoded. I have had a conversaiton with the template creator and he said that it is meant to be submissionDate, now changed.